### DISCUSSION TITLE
Nuxt 3 Tailwind some classes missing in the production

### DISCUSSION THREAD

**Discussion by hbilal9** [nuxt/nuxt#20348] [General]
When i upload my nuxt 3 app on ubuntu  aws ec2 after running npm run generate
when i goto dynamic page through router all things work correctly, but when i reload pages/restuarants/[username].vue
some of tailwind css classes missing,

**Comment by SherKhan32** [comment]
I am Facing the same Issue!

**Comment by ndragun92** [comment]
@hbilal9 if you are loading dynamic classes with JIT (just in time) enabled on the tailwind side you have to put classes in safelist classes.

[Safelist classes documentation](https://tailwindcss.com/docs/content-configuration#safelisting-classes)

Hope it helps

**Reply by SherKhan32** [reply]
@ndragun92 I tried this but our problem still wasn't solved.

Before Refresh SS:
![image](https://user-images.githubusercontent.com/79793926/233054843-c10e2e2b-b0fd-4cba-ab7d-e2d02c1f25a8.png)

After refresh SS:

![image](https://user-images.githubusercontent.com/79793926/233055170-a4c29b47-4220-4e78-8176-c76381ab9d5a.png)

**Reply by ndragun92** [reply]
@SherKhan32 do you have some hydration issues maybe?
Have you tried wrapping certain parts of the code in `<client-only></client-only>` tag and see if that solves it?

**Reply by hbilal9** [reply]
thank you so much it's working

**Reply by SherKhan32** [reply]
Thank you so much for your help! Your solution worked perfectly and saved me a lot of time and frustration. I really appreciate your willingness to share your expertise and help others in the community. Thanks again!

**Reply by ndragun92** [reply]
@SherKhan32 glad to hear that is solved 😊

**Reply by ndragun92** [reply]
@SherKhan32 Guess u can close the thread now 😊

**Reply by SherKhan32** [reply]
Sure 😊
